### PR TITLE
chore: add prop to toggle avatar animation on enter

### PR DESCRIPTION
**Pull request by PedroMiguelSS**
Adds `animateOnEnter` prop to `<Avatar>` component so that we can activate/deactivate image fade animation.

### CODE REVIEW

File: src/components/avatar/Avatar.js [hunk 1]
```diff
@@ -41,12 +41,14 @@ const Avatar = ({ className, name, image, preloadImage, ...rest }) => {
 Avatar.propTypes = {
     name: PropTypes.string,
     image: PropTypes.string,
-    preloadImage: PropTypes.bool,
     className: PropTypes.string,
+    preloadImage: PropTypes.bool,
+    animateOnEnter: PropTypes.bool,
```
Comment (satazor) [inline src/components/avatar/Avatar.js]: Perhaps rename to `animateImageOnEnter`?

File: src/components/avatar/Avatar.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ const Avatar = ({ className, name, image, preloadImage, ...rest }) => {
             <PreloadImage
                 src={ image }
                 decode={ preloadImage }
-                container={ imageContainer }
+                container={ animateOnEnter ? imageContainer : undefined }
```
Comment (satazor) [inline src/components/avatar/Avatar.js]: If `animateOnEnter` is true, shouldn't we simply change the `appear` prop on `<TransitionGroup>` instead?

**Review by satazor — CHANGES_REQUESTED** [review]

File: src/components/avatar/Avatar.js (hunk 2 above)
Comment (PedroMiguelSS) [reply src/components/avatar/Avatar.js]: I can do that. But do you prefer to pass the `animateOnEnter` prop to `imageContainer` function as an argument?

Comment (satazor) [reply src/components/avatar/Avatar.js]: Yep and it would map to appear.

**Review by satazor — APPROVED** [review]

**Pull request merged**